integrationtest/src/test/resources/fullFeatureTest/src/test/java/org/mapstruct/itest/simple/AnimalTest.java

```diff
@@ -0,0 +1,72 @@
+/**
+ *  Copyright 2012-2015 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.test.ignore;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import org.junit.Test;
+
+import org.mapstruct.ap.test.ignore.AnimalMapper;
+import org.mapstruct.ap.test.ignore.Animal;
+import org.mapstruct.ap.test.ignore.AnimalDto;
+
+/**
+ * Test for ignoring properties during the mapping.
+ *
+ * @author Gunnar Morling
+ */
+public class AnimalTest {
```

Comment:
You don't recognize it? :grin: 
I copied it from the processor tests, to have at least one check on a generated mapper.